docs/rfcs/1013-parsec-auth-scheme.md

```diff
@@ -0,0 +1,267 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# Schema for parsec-auth
+
+## Overview
+
+This RFC defines the schema used for communicating with the parsec-auth service.
+
+## Goals and Non-Goals
+
+The goal is to define the required schema used when communicating with the parsec-auth service.
+For simplicity, we will skip the fido2 part that will be defined in a later RFC.
+
+## Design
+
+### Account creation
+
+To start the account creation, the client start by sending its email.
+
+```yml
+{
+  "cmd": "account_creation_step1_email_validation",
+  "req": {
+    "email": "Email"
+  },
+  "reps": [
+    {
+      "status": "ok"
+    },
+    {
+      "status": "invalid_email"
+    }
+  ]
+}
+```
+
+On `ok`, the server would have sent a mail with a unique token used for next the request used to register the authentication method:
+
+```yml
+{
+  "cmd": "account_creation_step2_registration",
+  "req": {
+    "email_validation_token": "EmailValidationToken",
+    "account_manifest": "EncryptedAccountManifest",
+    "auth_medium": "AuthMedium"
+  },
+  "reps": [
+    {
+      "status": "ok",
+      "auth_medium_id": "AuthMediumID"
```

Comment:
tl;dr: remove this field

Long version:

auth medium ID should either be defined client-side or be returned by the server

Since we store the auth medium ID in the encrypted `AccountManifest`, we have no choice but to have have auth medium ID be defined client-side. And so returning auth medium ID to the client is useless.

Reply:
done